contrib/common.sh

```diff
@@ -5,10 +5,12 @@ function allow_vendor_change() {
 }
 
 function add_common_repos() {
-  zypper -q ar -f http://download.opensuse.org/repositories/devel:/languages:/perl/openSUSE_Leap_42.1/devel:languages:perl.repo
+  zypper -q rr systemsmanagement-chef
+  zypper -q rr systemsmanagement-puppet
   zypper -q ar -f http://download.opensuse.org/repositories/OBS:/Server:/Unstable/openSUSE_42.1/OBS:Server:Unstable.repo
   zypper -q ar -f http://download.opensuse.org/repositories/openSUSE:/Tools/openSUSE_42.1/openSUSE:Tools.repo
   zypper -q --gpg-auto-import-keys refresh
+  zypper -q -n dup -l --replacefiles
```

Comment:
@hennevogel This seems to have shaken up things a bit. When doing a vagrant setup from scratch there are a couple of issues popping up.
See https://gist.github.com/evanrolfe/ec958076657711c7d9c4206922f72c5f

This is caused by /etc/alternatives/rspec linking to a non-existing rspec binary. zypper in -f ruby2.1-rubygem-rspec-core would workaround this, but seems not to be a good solution to me.

And even if we go with this, there are more problems ahead:
```bash
vagrant exec rspec
/usr/lib64/ruby/2.1.0/rubygems/core_ext/kernel_require.rb:55:in `require': cannot load such file -- nyan_cat_formatter (LoadError)
```
And after silencing nyancat:
```bash
vagrant exec rspec
/usr/lib64/ruby/2.1.0/rubygems/core_ext/kernel_require.rb:55:in `require': cannot load such file -- bundler/setup (LoadError)
```
Note that I ran bundle install before.